nixos/modules/services/mail/stalwart-mail.nix

```diff
@@ -41,6 +48,20 @@ in {
       storage.blob = mkDefault "blob";
       resolver.type = mkDefault "system";
       resolver.public-suffix = mkDefault ["https://publicsuffix.org/list/public_suffix_list.dat"];
+    } // lib.mkIf (stalwartGreaterOrEqualThan "0.7.0") {
```

Comment:
Is there a pre 0.7.0 that can be installed via nixpkgs after that pr?
If no I don't understand why this logic is needed.

Reply:
still trying to figure out how to use the module for both versions, 0.6 and >=0.7. not yet finished :)